brayns/common/types.h

```diff
@@ -79,27 +79,27 @@ class Brayns;
 class ActionInterface;
 
 class Engine;
-typedef std::shared_ptr<Engine> EnginePtr;
+using EnginePtr = std::shared_ptr<Engine>;
```

Comment:
These changes could have been part of a different PR as it's a refactor of non clip plane specific code.

Reply:
That's called continuous refactoring :)